ctapipe/tools/tests/test_train.py

```diff
@@ -16,6 +16,12 @@ def test_train_particle_classifier(particle_classifier_path):
     ParticleClassifier.read(particle_classifier_path)
 
 
+def test_train_disp_reconstructor(disp_reconstructor_path):
```

Comment:
Maybe for another PR: this would better be called `test_read_trained_{}` ? The training happens in the fixture `{}_path`.